src/ssl.c

```diff
@@ -27631,7 +27673,7 @@ WOLFSSL_RSA *wolfSSL_d2i_RSAPublicKey(WOLFSSL_RSA **r, const unsigned char **pp,
 int wolfSSL_i2d_RSAPublicKey(WOLFSSL_RSA *rsa, const unsigned char **pp)
 {
     byte *der;
-    int derLen;
+    int derLen = 126;
```

Comment:
Is there a reason that "126" was used here?  Can it be set to a macro/define that is more descriptive?

Reply:
I missed to fix merge conflict.
I don't need initialization.